src/components/engineService.ts

```diff
@@ -261,10 +268,42 @@ export class EngineService {
 			shell: true,
 		});
 
-		childProcess.stderr.on('data', (data) => {
-			console.error(data.toString());
+		const errorMessage = new Set<string>();
```

Comment:
`errorMessages`, also it can be just a mutable array

Reply:
since the backend sometimes emits the same message we want to make sure we have unique strings